integration/flink/src/main/java/org/apache/carbon/flink/CarbonLocalWriter.java

```diff
@@ -168,7 +168,7 @@ public void commit() throws IOException {
       try {
         String stageInputPath = CarbonTablePath.getStageDir(
             table.getAbsoluteTableIdentifier().getTablePath()) +
-            CarbonCommonConstants.FILE_SEPARATOR + UUID.randomUUID();
+            CarbonCommonConstants.FILE_SEPARATOR + System.currentTimeMillis() + UUID.randomUUID();
```

Comment:
What is this required?

Reply:
@jackylk added

Reply:
@Zhangshunyu Can you explain why you need to make the file "ordered by time"